src/services/expenses.service.js

```diff
@@ -0,0 +1,79 @@
+const { createId } = require('../utils/id.service');
+
+let expenses = [];
+
+const resetExpenses = () => {
+  expenses = [];
+};
+
+const getExpenses = () => {
+  return expenses;
+};
+
+const getExpense = (id) => {
+  return expenses.find((elem) => elem.id === id);
```

Comment:
Ensure that the `id` parameter is consistently treated as a number throughout the codebase. Consider using `+id` here to explicitly convert it to a number, as done in other parts of the code.